src/onboarding/steps.ts

```diff
@@ -243,6 +264,22 @@ export function _getStepInfo({ firstScreenInStep, navigator, dispatch, props }:
           }
         },
       }
+    case Screens.SignInWithEmail:
+      return {
+        next: () => {
+          if (showRecoveryPhraseEducation) {
+            navigate(Screens.AccountKeyEducation, {
+              origin: 'cabOnboarding',
+            })
```

Comment:
I think we should remove this case and directly navigate to this screen from SignInWithEmail. Otherwise there is an edge case where our step counter fn could crash when it walks thru graph since `AccountKeyEducation` does not have a next step. Although practically this may never happen since `showRecoveryPhraseEducation` will be `false`, but anyways good to keep this clean.